infra/src/main/resources/exception/exceptionInfo.json

```diff
@@ -0,0 +1,43 @@
+[
```

Comment:
@r00ta Should we move this into the Manager and put a blank file in the Shard that we will add to as we do the work in https://issues.redhat.com/browse/MGDOBR-114 ?

Most of these errors only really make sense in the context of the Fleet Manager and it would make things clearer IMHO if each component owned it's own `exceptionInfo.json`. WDYT?

Reply:
@robpblake I thought the shard and the manager would have shared some exceptions. Or should we just duplicate the exception/error definition in every component?

Reply:
If error API is going to be available for shard and manager, then we need two files. 
If this is going to be exposed only in the manager, then just one file 
I hope you like this rule ;)

Reply:
@fjtirado the `errorAPI` is only in the manager. The user is not supposed to be able to reach directly any shard by design

Reply:
Then, all errors should be defined in just one file. It does not care that the error is not produced directly by the manager, it is handled by it anyway.